.github/workflows/benchmark_x86.yml

```diff
@@ -182,44 +148,90 @@ jobs:
           pattern: results_*
           path: ./results
           merge-multiple: true
-      
+
       - name: Set up the environment
         run: |
-          sudo apt-get update && sudo apt-get install -y yq
+          sudo apt-get update && sudo apt-get install -y yq jq # Added jq
```

Comment:
Remove "# Added jq". If you want to highlight the two installed packages, give each a line.